database/factories/ModelFactory.php

```diff
@@ -46,6 +45,48 @@
    ];
 });
 
+// precondition: RecordIssuerType must exist
+$factory->defineAs(App\RecordIssuer::class, RecordIssuerType::BILLORG_TYPE_NAME , function(Faker\Generator $faker){
+    $organization_name = $faker->unique()->randomElement(array(
+        "AIA Singapore Pte Ltd", "Boston Business School Pte Ltd",
+        "Singtel", "Starhub", "M1 Limited",
+        "Tampines Town Council","IRA",
+        "UOB Card Centre", "Citibank Credit Cards", "DBS Credit Cards","OCBC Credit Cards","Maybank Credit Cards","HSBC Credit Cards",
+        "OCBC Plus", "Maybank Kim Eng Securities Pte Ltd",
+        "UOB Car Financing Payment"
+    ));
+      return [
+          'name' => $organization_name,
+          'type' => RecordIssuerType::type( RecordIssuerType::BILLORG_TYPE_NAME)->first()->id
+      ];
+});
+
+// precondition: RecordIssuerType must exist
+$factory->defineAs(App\RecordIssuer::class,RecordIssuerType::BANK_STATEMENT_TYPE_NAME , function(Faker\Generator $faker){
+    $organization_name = $faker->unique()->randomElement(array(
+        "POSB Savings",
+        "Citibank Interestplus" ,
+        "Maybank Savings" ,
+        "UOB Passbook Sasvings",
+        "DBS Coporate"
+    ));
+    return [
+        'name' => $organization_name,
+        'type' => RecordIssuerType::type(RecordIssuerType::BILLORG_TYPE_NAME)->first()->id
+    ];
+});
+
+$factory->defineAs(App\Record::class,RecordIssuerType::BILLORG_TYPE_NAME, function(Faker\Generator $faker){
+    $issue_date = Carbon::createFromTimeStamp($faker->dateTimeBetween($start_date = '- 5 years', $end_date = 'now')->getTimestamp());
```

Comment:
This is `RecordIssuerType,` not `Record.` What you're writing is for `Record,` which has already been written.